src/builder/Actions.sol

```diff
@@ -477,6 +483,99 @@ library Actions {
         return (quarkOperation, action);
     }
 
+    struct BorrowInput {
+        Accounts.ChainAccounts[] chainAccountsList;
+        uint256 amount;
+        string assetSymbol;
+        uint256 blockTimestamp;
+        address borrower;
+        uint256 chainId;
+        uint256[] collateralAmounts;
+        string[] collateralTokenSymbols; // collateralAssetSymbols?
```

Comment:
I think we've been using `token` and `asset` interchangeably. I went with `collateralToken` to match the `BorrowActionContext` naming